config/credentials.default.coffee

```diff
@@ -2,8 +2,8 @@ module.exports = (options) ->
   kiteHome = "$KONFIG_PROJECTROOT/generated/kite_home/koding"
 
   kodingdev_master =
-    accessKeyId: "AKIAJYFSB4KOBWODAHEQ"
-    secretAccessKey: "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"
```

Comment:
they're in a public git history now, don't forget to recycle these keys. cc/ @hakankaradis @szkl in case they are ours